android/src/main/java/com/reactnativefacetec/Processors/EnrollmentProcessor.java

```diff
@@ -0,0 +1,178 @@
+//
+// Welcome to the annotated FaceTec Device SDK core code for performing secure Enrollment!
+//
+package com.reactnativefacetec.Processors;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.Looper;
+import android.util.Log;
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.IOException;
+
+//import com.facetec.sampleapp.SampleAppActivity;
+import com.facetec.sdk.*;
+
+// This is an example self-contained class to perform Enrollment with the FaceTec SDK.
+// You may choose to further componentize parts of this in your own Apps based on your specific requirements.
+
+// Android Note 1:  Some commented "Parts" below are out of order so that they can match iOS and Browser source for this same file on those platforms.
+// Android Note 2:  Android does not have a onFaceTecSDKCompletelyDone function that you must implement like "Part 10" of iOS and Android Samples.  Instead, onActivityResult is used as the place in code you get control back from the FaceTec SDK.
+public class EnrollmentProcessor extends Processor implements FaceTecFaceScanProcessor {
```

Comment:
These extra processors aren't needed correct? We just need the LivenessProcessor. Is this one being used?